marlowe-playground-client/package.json

```diff
@@ -7,7 +7,7 @@
     "webpack:server": "webpack-dev-server --progress --inline --hot --mode=development --host 0.0.0.0 --display verbose",
     "webpack:server:debug": "DEBUG=purs-loader* DEBUG_DEPTH=100 webpack-dev-server --progress --inline --hot",
     "purs:compile": "spago build",
-    "purs:ide": "purs ide server --log-level=debug 'src/**/*.purs' 'generated/**/*.purs' 'test/**/*.purs' '../web-common/**/*.purs'",
+    "purs:ide": "purs ide server --log-level=debug 'src/**/*.purs' 'generated/**/*.purs' 'test/**/*.purs' './web-common/**/*.purs'",
```

Comment:
`'./web-common/**/*.purs'` could just be `'web-common/**/*.purs'`

Reply:
you are right, that is equivalent - is there any benefit that the one has over the other though?

Reply:
never mind - i'll just change it everywhere.